cmd/inferconfig/testdata/expected/circleci-demo-python-django.yml

```diff
@@ -1,5 +1,5 @@
 # This config was automatically generated from your source code
-# Stacks detected: deps:python:.,package_manager:pipenv:.
+# Stacks detected: deps:python:.,file:manage.py:.,package_manager:pipenv:.
```

Comment:
Maybe I'd call this e.g "framework:django" instead of "manage.py"

Reply:
That is what I had before 😅 But then saw flask projects also containing that file https://github.com/CircleCI-Public/circleci-demo-python-flask. Can make the change if we're fine with labelling other frameworks that have manage.py django

Reply:
Aha! `file:manage.py` is fine then, I'll leave the decision to you.